pkg/onit/common.go

```diff
@@ -51,6 +51,12 @@ const (
 	OnosAll NodeType = "all"
 )
 
+type Port int
```

Comment:
FWIW I don't think we need to define a new type here considering ports are never even programmatically specified (only through the CLI) so the type won't even be used. Not something that needs to be changed, just noting why I don't think it's needed.

Reply:
agreed. I wasn't sure which one is better. Will define it as int.